build

```diff
@@ -7,6 +7,12 @@ GO="godep go"
 ORG_PATH="github.com/kelseyhightower"
 REPO_PATH="${ORG_PATH}/confd"
 
+export GOPATH=${PWD}/gopath
+
+rm -f $GOPATH/src/${REPO_PATH}
```

Comment:
you are right. line 10 should be replaces with something like this:

SCRIPT=`realpath $0`
SCRIPTPATH=`dirname $SCRIPT`

export GOPATH=$SCRIPTPATH/gopath


what do you think?